tests/ruby/RubySmellBearTest.py

```diff
@@ -0,0 +1,11 @@
+from bears.ruby.RubySmellBear import RubySmellBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+good_file = ('',)
```

Comment:
Not a great testcase IMO ... Possibly something better ? Or make an issue

PS: The linter may just give no output for an empty file and so on.